assets/js/components/alerts/AlertIndexTable.jsx

```diff
@@ -1,28 +1,86 @@
-import React, { Fragment, useState } from 'react';
+import React, { Fragment } from 'react';
 import { Link } from 'react-router-dom';
-import { Table } from 'antd';
+import { Table, Button } from 'antd';
 import Text from 'antd/lib/typography/Text';
+import DeviceGroupTriggerIcon from '../../../img/alerts/alert-trigger-device-group.svg';
+import FunctionTriggerIcon from '../../../img/alerts/alert-trigger-function.svg';
+import IntegrationTriggerIcon from '../../../img/alerts/alert-trigger-integration.svg';
+import moment from 'moment';
+import { DeleteOutlined } from '@ant-design/icons';
+import UserCan from '../common/UserCan'
 
-export default () => {
-  const [selectedRows, setSelectedRows] = useState([]);
-  const [allSelected, setAllSelected] = useState(false);
+export default (props) => {
+  const renderTrigger = (trigger) => {
+    switch (trigger) {
+      case 'device/group':
+        return (
+          <span><img src={DeviceGroupTriggerIcon} style={{ height: 20, marginRight: 5 }} />Device/Group</span>
+        );
+      case 'function':
```

Comment:
do we have function triggers already for alerts? or is this just placeholder

Reply:
we have none that have been defined by prod yet so just a placeholder for now